templates/cloudflare-workers/remix.config.js

```diff
@@ -1,11 +1,8 @@
 /** @type {import('@remix-run/dev').AppConfig} */
 module.exports = {
-  serverBuildTarget: "cloudflare-workers",
-  server: "./server.js",
+  serverModuleFormat: "esm",
+  serverBuildPath: "build/index.mjs",
   devServerBroadcastDelay: 1000,
+  devServerPort: 8002,
   ignoredRouteFiles: ["**/.*"],
-  // appDirectory: "app",
-  // assetsBuildDirectory: "public/build",
-  // serverBuildPath: "build/index.js",
-  // publicPath: "/build/",
```

Comment:
We keep this commented out for convenience in our other templates, so maybe we should keep them here as well?